database/migrations/2018_12_21_175943_create_ama_announcements_table.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateAmaAnnouncementsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('ama_announcements', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('title');
+            $table->text('text');
+            
+
+            $table->timestamps();
+        });
+
+        Schema::table('ama_announcements', function($table) {
```

Comment:
No need to this separately - this whole part can come under `$table->text('text');` in the `Schema::create` part. Like this:

```
Schema::create('ama_announcements', function (Blueprint $table) {
    $table->increments('id');
    $table->string('title');
    $table->text('text');
    $table->unsignedInteger('user_id');
    $table->foreign('user_id')->references('id')->on('users');
    $table->timestamps();
});
```

Please adjust this and the `amas` migration below.